code-challenges.rb

```diff
@@ -31,3 +45,13 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+# def palindrometer (string_array)
+#     if string_array.downcase == string_array.downcase.reverse
+#         p "#{string_array} is a palindrome"
+#     else
+#         p "#{string_array} is not a palindrome"
+#     end
+# end
+
+palindrometer(is_palindrome2)
```

Comment:
Nice job on the logic. Make sure you call your method multiple times. Each with one test variable.